m4_sessions/1708-inning/group_two.md

```diff
@@ -1,9 +1,45 @@
   # Group Two
 
 ### Session Title:
+* The Psychology of UX/UI
 
 ### Session Outcomes: 
+What do we want people attending the session to learn or walk away with at the end of the session?
+
+#### Education
+* Understanding the difference of UX and UI
+* The importance of NOT ignoring UX/UI (why *everyone* should care about it)
+* Understanding the emotions of UX/UI
+
+#### Simplicity
+* The power of simplicity
+* Understanding design patterns in UX/UI (and recognizing/understanding bad or anti-patterns)
+
+#### In Your Career
+* Understanding proper usage of micro-animations (and NOT overusing the BLINK tag) - live examples/demo
+* How to incorporate UX/UI thinking into your career as a developer
+* Being more actively aware of usability
 
 ### Session Outline:
+Intro to UX/UI (10 minutes)
+- need to come up with what this looks like
+Design Thinking Video (10 minutes or less)
+- need to find one or two great videos
+Leader Group Exercise (10 minutes)
+- what we think is great/bad design with examples
+Small Group Exercise (10 minutes of research/talking, 5 minutes of sharing)
+- need to come up with "what" the groups will be doing
+Share principles of Simplicity, w/ slide show (10 minutes)
+Career Prep (15 minutes)
 
 ### Names of group members:
+* Dave Weinstock
+* Juan Jimenez
+* Cole Worsley
+* Justyna Field
+* Jason Collins
+
+### Resources (how to level up)
+* (book) Don't Make Me Think
```

Comment:
I'd recommend divide and conquer to determine the final resources you want to use in the session.

Reply:
Good feedback, we'll do that. 👍